### PR TITLE
Standardize on capitalization of windows.h fix #687.

### LINKED ISSUE #687: Standardize on capitalization of windows.h
Semi-related to https://github.com/Microsoft/WinObjC/pull/634

With us cross-compiling tests on OSX, we need to watch out for capitalization of file names, which are case-sensitive on OSX. One noticeable outlier is windows.h, which has 44 instances of lowercase and 27 instances of uppercase (though only 2 in unit tests). We ought to standardize this for safety purposes.

### PR DESCRIPTION
**Pull request by msft-Jeyaram**

### CODE REVIEW

**Comment by DHowett-MSFT** [comment]
:shipit:

**Comment by rajsesh-msft** [comment]
:shipit:

**Comment by msft-Jeyaram** [comment]
In CI Build

**Comment by ms-jihua** [comment]
:shipit:

File: Frameworks/CoreLocation/CLGeocoder.mm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 //
 //******************************************************************************
 
-#import "Windows.h"
+#import "windows.h"
```
Comment (aballway) [inline Frameworks/CoreLocation/CLGeocoder.mm]: >"windows.h" [](start = 8, length = 11)

Can we also standardize <> vs ""?

Comment (msft-Jeyaram) [inline Frameworks/CoreLocation/CLGeocoder.mm]: Given it's all over the place, for (W/w)indows. it would be a change on it's own.

**Pull request merged**